### ISSUE TITLE
Comments on "Manipulating holograms using the MRTK Hand Physics service"

### ISSUE THREAD

**Issue by LocalJoost** [LocalJoost/BlogComments#390]
Original article: https://localjoost.github.io/Manipulating-holograms-using-the-MRTK-Hand-Physics-service/

**Comment by raggnic** [comment]
hi joost
that's pretty nice. Do you know if this service still exists in MRTK3 in another form?

**Comment by LocalJoost** [comment]
@raggnic nope, but fortunately [some idiot with too much time on his hands 😁](https://github.com/[LocalJoost/QuestBouncer](https://github.com/LocalJoost/QuestBouncer)) has built something like that. It's called the Hand Smash service, is cross-platform, and you can find it in [this project](https://github.com/LocalJoost/QuestBouncer)

**Comment by raggnic** [comment]
cool that's inspiring! Thank you for sharing

**Comment by LocalJoost** [comment]
Thanks, and

![youre-welcome-welcome](https://github.com/user-attachments/assets/657ee109-7155-4da4-8f97-848bb5e4bdfc)
